trulens/nn/models/pytorch.py

```diff
@@ -284,7 +286,23 @@ def fprop(
 
                 batched_model_args.append(val)
 
+            # tile batches for interpolation
+            for k, val in model_kwargs.items():
+                doi_resolution = int(doi_repeated_batch_size / val.shape[0])
+                tile_shape = [1 for _ in range(len(val.shape))]
+                tile_shape[0] = doi_resolution
+                repeat_shape = tuple(tile_shape)
+
+                if isinstance(val, np.ndarray):
+                    val = np.tile(val, repeat_shape)
+
+                elif torch.is_tensor(val):
+                    val = val.repeat(repeat_shape)
+
+                batched_model_kwargs[k] = val
```

Comment:
nit: let's put this in a shared function that both can call.

Reply:
Done.